Units/parser-cxx.r/bug1799343-1.cpp.d/expected.tags

```diff
@@ -1,6 +1,7 @@
 C	input.cpp	/^struct C { int x; };$/;"	s	file:
-D	input.cpp	/^ D() { x = 123; }$/;"	f	struct:D
+x	input.cpp	/^struct C { int x; };$/;"	m	struct:C	file:
 D	input.cpp	/^struct D : ::C {$/;"	s	file:
+D	input.cpp	/^ D() { x = 123; }$/;"	f	struct:D	file:
```

Comment:
file: is attached, too. (I don't say this is a bug. Just specifying the incompatibility.)